src/collective/jekyll/browser/collection_view.pt

```diff
@@ -44,38 +43,45 @@
                                                 obj_wf_state obj/review_state|python: wtool.getInfoFor(obj, 'review_state', '');
                                                 obj_wf_state_class python:'state-' + normalizeString(obj_wf_state);
                                                 obj_url python:url+'/view' if obj_type in use_view_action else url;
                                                 title_or_id obj/pretty_title_or_id">
                             <tal:block define="value python: getattr(obj, field, None)">
-                                <td tal:condition="python: field != 'Title'"
-                                    tal:content="python:here.formatCatalogMetadata(value)"
+                                <td tal:condition="python: field != 'Title' and field not in ['ModificationDate',
+                                        'CreationDate', 'EffectiveDate',
+                                        'ExpirationDate', ]"
+                                    tal:content="value"
                                     i18n:translate=""/>
+                                <td tal:condition="python:field in ['ModificationDate',
+                                        'CreationDate', 'EffectiveDate',
+                                        'ExpirationDate', ]">
+                                    <span tal:replace="python:toLocalizedTime(value,long_format=1)">
+                                      August 16, 2001 at 23:35:59
+                                    </span>
+                                </td>
                                 <td tal:condition="python: field == 'Title'">
-                                    <a href="#" tal:attributes="href obj_url;
-                                                                title obj_description;">
-                                        <img src="#"
-                                             height="16"
-                                             width="16"
-                                             tal:condition="obj_icon/url"
-                                             tal:attributes="src obj_icon/url;
-                                                             alt obj_icon/description;
-                                                             title obj_icon/title;
-                                                             width obj_icon/width;
-                                                             height obj_icon/height;"/>
+                                    <a href="#"
+                                        tal:condition="obj_has_image"
+                                        tal:attributes="href obj_url;
+                                                        title obj_description;">
+                                        <img class="thumb-icon"
+                                            tal:define="getIcon python:obj.getURL() +'/@@images/image/icon'"
```

Comment:
`python:` in the template is often a smell. I would compute the image URL in the view class.